xbmc/windowing/gbm/WinSystemGbmEGLContext.cpp

```diff
@@ -11,6 +11,7 @@
 #include "OptionalsReg.h"
 #include "cores/VideoPlayer/DVDCodecs/DVDFactoryCodec.h"
 #include "cores/VideoPlayer/VideoRenderers/RenderFactory.h"
+#include "drm/DRMAtomic.h"
```

Comment:
This breaks the abstraction.

It would be nice if we could check the next layer up `if (supportsFences())` in CWinSystemGbm which then calls into CDRMUtils of the same call. The atomic and legacy version can then provide overrides if they support using fences or not.